openedx/core/djangoapps/credit/email_utils.py

```diff
@@ -180,3 +186,32 @@ def _email_url_parser(url_name, extra_param=None):
     dashboard_url_path = reverse(url_name) + extra_param if extra_param else reverse(url_name)
     dashboard_link_parts = ("https", site_name, dashboard_url_path, '', '', '')
     return urlparse.urlunparse(dashboard_link_parts)
+
+
+def _get_credit_providers(user, course_key):
```

Comment:
I would suggest moving this method to `openedx/core/djangoapps/commerce/utils.py` and changing its name to `get_credit_providers`.